site/_data/docs/extensions/toc.yml

```diff
@@ -51,6 +51,7 @@
     - url: /docs/extensions/mv3/service_workers/basics
     - url: /docs/extensions/mv3/service_workers/service-worker-lifecycle
     - url: /docs/extensions/mv3/service_workers/events
+    - url: /docs/extensions/mv3/tut_websockets
```

Comment:
```suggestion
    - url: /docs/extensions/mv3/tut-websockets
```
NIT 😇

Reply:
I very much agree with this, but all other tutorials use an underscore as well...

<img width="621" alt="Screenshot 2023-07-12 at 15 36 19" src="https://github.com/GoogleChrome/developer.chrome.com/assets/380472/aa06651f-efe4-4953-8308-1018330cc414">